src/style.css

```diff
@@ -0,0 +1,78 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.form_search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.icon_2 {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
```

Comment:
avoid numbers, `first`, `second` in names. It is not clear.
Use `--small` modifier instead